[RHACS] Patch release notes for 4.5.3 (#82687)
openshift/openshift-docs: release_notes/45-release-notes.adoc

* https://access.redhat.com/security/cve/CVE-2024-3727[CVE-2024-3727]: Fixed a vulnerability related to Scanner where the `containers/image` digest type did not guarantee a valid type.

[id="about-release-453_{context}"]
== About release 4.5.3

*Release date*: 30 September 2024

This release of {product-title-short} includes the following changes:

> * Fixes a broken pipe error that caused the Central UI dashboard to display incomplete data.

Reviewer: ```suggestion
* Fixed a broken pipe error that caused the Central user interface (UI) dashboard to display incomplete data.
```

> * Adds a new `--with-database-only` option to the `roxctl central debug download-diagnostics` command. Use it to generate diagnostic bundles for troubleshooting connection issues related to policy violations and deployments.

Reviewer: ```suggestion
* Added the `--with-database-only` option to the `roxctl central debug download-diagnostics` command. You can use it to generate diagnostic bundles for troubleshooting connection issues related to policy violations and deployments.
```

This release of {product-title-short} fixes the following security vulnerability:

> * https://access.redhat.com/security/cve/CVE-2024-39249[CVE-2024-39249]: Fixed a Regular expression denial of service (ReDoS) vulnerability in {product-title-short} main container.

Reviewer: ```suggestion
* https://access.redhat.com/security/cve/CVE-2024-39249[CVE-2024-39249]: Fixed a Regular expression denial of service (ReDoS) vulnerability in the {product-title-short} main container.
```
